plasmapy/mathematics/mathematics.py

```diff
@@ -157,3 +158,65 @@ def plasma_dispersion_func_deriv(zeta):
     Zprime = -2 * (1 + zeta * plasma_dispersion_func(zeta))
 
     return Zprime
+
+
+def Fermi_integral(x, j):
+    r"""Calculate the complete Fermi-Dirac integral.
+    j is the index
+    x is the argument
+
+    Parameters
+    ----------
+    x : Quantity
+        Argument of the Fermi-Dirac integral function.
+
+    j : Quantity
+        Order of the Fermi-Dirac integral function.
+
+    Returns
+    -------
+    integral : Quantity, complex
+        Complete Fermi-Dirac integral for given argument and order.
+
+    Raises
+    ------
+    TypeError
+        If the argument is invalid.
+    UnitsError
+        If the argument is a Quantity but is not dimensionless
+    ValueError
+        If the argument is not entirely finite
+
+    See also
+    --------
+
+
+    Notes
+    -----
+    The complete Fermi-Dirac integral is defined as:
+
+    .. math::
+        F_j (x) = \frac{1}{\Gamma (j+1)} \int_0^\inf \frac{t^j}{e^{t-x} + 1} dt
+        for j > 0
+
+    This is equivalent to the following polylograithm function:
```

Comment:
typo: polylog**ra**ithm. Also, could be worth adding:

    where j is the polylogarithm's order or index.

But maybe that's clear enough from context.

Reply:
fixed typo. I think it is clear enough from context. Also j is not the order of the polylog, j+1 is the order for the polylog, whereas j is the order of the Fermi integral.